quinn-proto/src/connection/mod.rs

```diff
@@ -1337,7 +1337,7 @@ impl Connection {
         let loss_delay = cmp::max(rtt.mul_f32(self.config.time_threshold), TIMER_GRANULARITY);
 
         // Packets sent before this time are deemed lost.
-        let lost_send_time = now - loss_delay;
+        let lost_send_time = now.checked_sub(loss_delay).unwrap();
```

Comment:
Seems fine to make this explicit? Seems pretty obviously safe -- we wouldn't worry about `loss_delay` overflowing a contemporary `Instant`.

Reply:
:+1: